protected/controllers/DayThreeController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+class DaythreeController extends Controller
+{
+
+    public function actionAbout()
+    {
+        $this->render('about');
+    }
+
+    public function actionError()
+    {
+        if ($error = Yii::app()->errorHandler->error) {
+            if (Yii::app()->request->isAjaxRequest) {
+                echo $error['message'];
+            } else {
+                $this->render('error', $error);
+            }
+        }
+    }
+
+    public function actionKaprekar()
+    {
+        $kaprekar_numbers = array();
+        $index            = 0;
+
+        for ($base_num = 1; $base_num < 1000; $base_num++) {
+            $square_num = pow($base_num, 2);
+            $square_str = (string)$square_num;
+            $mid        = (int)strlen($square_str) / 2;
+            $left       = (int)substr($square_str, 0, $mid);
+            $right      = (int)substr($square_str, $mid);
+            $sum        = $left + $right;
+
+            if ($sum == $base_num) {

```

Comment:
I can see that you are already using `===` instead of `==`, which is good, so use it as much as possible.